.travis.yml

```diff
@@ -10,6 +10,7 @@ before_install:
   - curl ifconfig.co|xargs echo "Travis IP address is ";
 
 script:
+  - "mvn fmt:check verify"
```

Comment:
Probably don't need the verify because of the `- mvn test -B` below and there are no integration tests